utm.yaml

```diff
@@ -1271,15 +1279,78 @@ components:
         status:
           $ref: '#/components/schemas/UssAvailabilityStatus'
 
+    #
+    # Data structures for logging
+    #
+
+    USSLogSet:
+      type: object
+      description: |-
+        The set of log data fulfilling this standard's Logging requirements.
+      properties:
+        messages:
+          description: Outgoing messages sent to other USSs and the DSS, and incoming
+            messages received from other USSs, including instances where an expected
+            response to a request is not received.
+          type: array
+          items:
+            $ref: '#/components/schemas/ExchangeRecord'
+          default: []
+        operator_notifications:
+          description: Instances of operator notifications as specifically required
+            within this standard.
+          type: array
+          items:
+            $ref: '#/components/schemas/UserNotificationRecord'
+          default: []
+        operator_inputs:
+          description: Instances of operator input as specifically required within
+            this standard.
+          type: array
+          items:
+            $ref: '#/components/schemas/UserInputRecord'
+          default: []
+        operator_associations:
+          description: For a USS that manages operational intents, associations of
+            an operator with operational intents that transitioned to the Accepted
+            state.
+          type: array
+          items:
+            $ref: '#/components/schemas/OperatorAssociation'
+          default: []
+        planning_attempts:
+          description: For a USS that manages operational intents, instances where
+            an operational intent could not be planned or replanned due to conflicts
+            with other operational intents or constraints.
+          type: array
+          items:
+            $ref: '#/components/schemas/PlanningRecord'
+          default: []
+        operation_positions:
+          description: For a USS performing conformance monitoring, all position data
+            used for conformance monitoring that is ingested from the UA.
+          type: array
+          items:
+            $ref: '#/components/schemas/OperationRecord'
+          default: []
+        constraint_provider_associations:
+          description: For a USS that performs constraint management, associations
+            of an authorized constraint provider with all constraints that transition
+            to the valid state.
+          type: array
+          items:
+            $ref: '#/components/schemas/ConstraintProviderAssociation'
+          default: []
+
     ExchangeRecord:
       type: object
       description: Details of a request/response data exchange.
       required:
```

Comment:
Shouldn't 'problem' be a required field here as well? I think that if somebody is going to report an issue then they should provide as much detail as possible to help with the investigation.

Reply:
`USSLogSet.messages` is a list of `ExchangeRecord` and presumably most of those won't have any problems.  I would imagine that if an `ExchangeRecord` with no `problem` specified was submitted to an error reports endpoint, that endpoint would probably respond 400 since there was no problem reported.